shell/platform/darwin/ios/framework/Source/FlutterPlatformPlugin.mm

```diff
@@ -47,6 +60,9 @@ - (instancetype)initWithEngine:(fml::WeakPtr<FlutterEngine>)engine {
 
   if (self) {
     _engine = engine;
+    NSNumber* infoValue = [[NSBundle mainBundle]
```

Comment:
This will crash if someone puts the wrong key type in their Info.plist. Since that's a developer error that's probably okay, but it might be nice to check the type and log (in debug only) a clear error message if it's wrong.

Reply:
done